packages/jest/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [8.0.0]
```

Comment:
This changelog release entry is empty. Some others are as well. This looks like a mistake.

Reply:
The `8.0.0` release entry is empty for all packages except the base config, because only the base config changed. I agree that it may leave a reader wondering what's going on. I created an issue for it here: https://github.com/MetaMask/action-create-release-pr/issues/71